jax/interpreters/pxla.py

```diff
@@ -769,15 +788,15 @@ def parallel_callable(fun: lu.WrappedFun,
         f"Specified axis_size {global_axis_size} doesn't match received "
         f"axis_size {axis_size}.")
 
-  if devices is not None and backend_name is None:
-    backend = xb.get_device_backend(devices[0])
+  if devices is not None and backend is None:
+    be = xb.get_device_backend(devices[0])
   else:
-    backend = xb.get_backend(backend_name)
+    be = xb.get_backend(backend)
```

Comment:
Sorry but this naming change is really weird and IMO significantly lowers the readability of this function